Sources/TwitterTextEditor/TextView.swift

```diff
@@ -387,4 +387,28 @@ final class TextView: UITextView {
         }
     }
 #endif
+
+    // MARK: - Delegate forwarding
```

Comment:
Format: `// MARK: - UIScrollView` would be better as pattern, since `delegate` is originally from `UIScrollView`.